langwatch/src/components/TraceToPlaygroundLink.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from "@chakra-ui/next-js";
+import { Button, Tooltip } from "@chakra-ui/react";
+import { projectRoutes } from "~/utils/routes";
+import { Image as ImageIcon } from "react-feather";
+import React from "react";
+/**
+ * Compose the link to the playground with the given project slug, trace ID, and
+ * span ID.
+ *
+ * The playground link is composed of the project slug, trace ID, and span ID in
+ * the following format:
+ *  /[projectSlug]/playground?traceId=[traceId]&span=[spanId]
+ *
+ * @param projectSlug - The project slug to link to.
+ * @param traceId - The trace ID to load in the playground.
+ * @param spanId - The span ID to load in the playground.
+ * @returns The link to the playground with the given trace and span.
+ */
+function useLinkHref(projectSlug: string, traceId: string, spanId: string) {
+  return React.useMemo(() => {
+    const queryString = new URLSearchParams({
+      traceId,
+      span: spanId,
+    }).toString();
+
+    const path = projectRoutes.playground.path.replace(
+      "[project]",
+      projectSlug
+    );
+
+    return `${path}?${queryString}`;
+  }, [projectSlug, traceId, spanId]);
+}
```

Comment:
this is a very manual way to do it, it's not the way we do elsewhere on the project, we have nextjs helpers for that instead, you can simply do:

```typescript
const router = useRouter();

void router.push(
  {
    pathname: `/${project?.slug}/playground`,
    query: {
      traceId,
      spanId
    },
  },
  undefined,
  { shallow: true }
);
```

Reply:
or straight up `href="url_here"`

Reply:
Sure. I have skipped the hook entirely.